scripts/release/marketplaceRelease.js

```diff
@@ -116,7 +120,7 @@ async function fetch(method, url, body) {
     try {
         response = await nodefetch(url, httpsOptions);
     } catch (error) {
-        console.error(error);
+        throw new Error(`An error occurred while retrieving data from ${url}`);
```

Comment:
Are we swallowing the node fetch error here? If so, it would be good to throw that higher up as well.

Reply:
Fixed